entity-service-attribute-translator/src/main/java/org/hypertrace/entity/attribute/translator/AttributeMetadata.java

```diff
@@ -0,0 +1,16 @@
+package org.hypertrace.entity.attribute.translator;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+@Data
+@EqualsAndHashCode
+public class AttributeMetadata {
```

Comment:
Naming: AttributeMetadata is the name of what''s being represented, this is just an identifier. Maybe `AttributeIdentifier` or `AttributeMetadataIdentifier`?